### PR TITLE
Enable ebs-csi driver on AWS, add region + kubernetes_version vars

### PR DESCRIPTION
**Pull request by iameskild**
~Fixes | Closes | Resolves #~ Related to: #1488

## Changes introduced in this PR:

- Add required `aws-ebs-csi-driver` on AWS for kubernetes v1.23+
  - https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
  - 1.23 is now the latest Kubernetes version
- Corollary to  #1493 
  - ensure that `region` and `kubernetes_version` are passed to tf scripts 

## Types of changes

What types of changes does your PR introduce?

_Put an `x` in the boxes that apply_

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

### Requires testing

- [ ] Yes
- [ ] No

### In case you checked yes, did you write tests?

- [ ] Yes
- [ ] No

## Documentation

Does your contribution include breaking changes or deprecations?
If so have you updated the documentation?

- [ ] Yes, docstrings
- [ ] Yes, main documentation
- [ ] Yes, deprecation notices

## Further comments (optional)

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered and more.

### CODE REVIEW

**Comment by viniciusdc** [comment]
LGTM, could you wan IT once this is merged into main?

**Comment by iameskild** [comment]
success :)
https://github.com/Quansight/qhub-integration-test/actions/runs/3248110219

**Review by viniciusdc — APPROVED** [review]
LGTM, Thanks @iameskild

**Pull request merged**
